src/CMakeLists.txt

```diff
@@ -386,6 +388,10 @@ foreach(TERRA_TEST ${TERRA_TESTS})
     COMMAND $<TARGET_FILE:TerraExecutable> ${TERRA_TEST}
     WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
   )
+  set_tests_properties(
+    ${TERRA_TEST} PROPERTIES
+    ENVIRONMENT "LUA_PATH=./?.lua"
```

Comment:
I wonder why this doesn't fail currently.

Reply:
That's a really good question isn't it 🙃